### PR TITLE
Mention mutual exclusivity of MUSIC and Arbor in documentation

### PR DESCRIPTION
**Pull request by sarakonradi**
This PR adds a note to the MUSIC documentation, describing the mutual exclusivity of MUSIC and the recording backend for Arbor. It informs the user that both cannot be enabled at the same time.

### CODE REVIEW

File: doc/tutorials/music_tutorial/music_tutorial_1.rst [hunk 1]
```diff
@@ -7,6 +7,10 @@ at runtime. It can be used to couple NEST with other simulators, with
 applications for stimulus generation and data analysis and visualization and
 with custom applications that also use the MUSIC interface.
 
+.. note::
+
+   Please note that MUSIC and Arbor are mutually exclusive and cannot be enabled at the same time.
```
Comment (jougs) [inline doc/tutorials/music_tutorial/music_tutorial_1.rst:12]:
```suggestion
   Please note that MUSIC and the recording backend for Arbor are mutually exclusive and cannot be enabled at the same time. 
```

File: doc/tutorials/music_tutorial/music_tutorial_2.rst [hunk 2]
```diff
@@ -1,5 +1,9 @@
 Connect two NEST simulations using MUSIC
-======================================================
+========================================
+
+.. note::
+
+   Please note that MUSIC and Arbor are mutually exclusive and cannot be enabled at the same time.
```
Comment (jougs) [inline doc/tutorials/music_tutorial/music_tutorial_2.rst:6]:
```suggestion
   Please note that MUSIC and the recording backend for Arbor are mutually exclusive and cannot be enabled at the same time. 
```

File: doc/tutorials/music_tutorial/music_tutorial_3.rst [hunk 3]
```diff
@@ -13,8 +13,11 @@ Let’s take a look at a pair of programs that send and receive spikes.
 These can be used as inputs or outputs to the NEST models we created
 above with no change to the code. C++ code tends to be somewhat
 longwinded so we only show the relevant parts here. The C++ interface is
-divided into a setup phase and a runtime phase. Let’s look at the setup
-phase first:
+divided into a setup phase and a runtime phase. You can see the setup below. 
+
+.. note::
+
+   Please note that MUSIC and Arbor are mutually exclusive and cannot be enabled at the same time.
```
Comment (jougs) [inline doc/tutorials/music_tutorial/music_tutorial_3.rst:20]:
```suggestion
   Please note that MUSIC and the recording backend for Arbor are mutually exclusive and cannot be enabled at the same time. 
```

File: doc/tutorials/music_tutorial/music_tutorial_4.rst [hunk 4]
```diff
@@ -13,6 +13,10 @@ description <music_tutorial_3>` for explanation. Below we will only highlight th
 differences to the C++ API. The full example code is in the
 ``pymusic`` directory in the MUSIC repository.
 
+.. note::
+
+   Please note that MUSIC and Arbor are mutually exclusive and cannot be enabled at the same time.
```
Comment (jougs) [inline doc/tutorials/music_tutorial/music_tutorial_4.rst:18]:
```suggestion
   Please note that MUSIC and the recording backend for Arbor are mutually exclusive and cannot be enabled at the same time. 
```

File: doc/tutorials/music_tutorial/music_tutorial_parrot.rst [hunk 5]
```diff
@@ -61,4 +61,9 @@ This is true for devices in general of course, but this connection
 pattern, where you want to record the MUSIC input from another
 simulation, is so common that it’s worth warning for this.
 
+.. note::
+
+   Please note that MUSIC and Arbor are mutually exclusive and cannot be enabled at the same time.
```
Comment (jougs) [inline doc/tutorials/music_tutorial/music_tutorial_parrot.rst:66]:
```suggestion
   Please note that MUSIC and the recording backend for Arbor are mutually exclusive and cannot be enabled at the same time. 
```

File: doc/tutorials/music_tutorial/music_tutorial_setlatency.rst [hunk 6]
```diff
@@ -30,4 +30,9 @@ in our example, MUSIC does not need to use any of that extra delay so
 things may work fine, but in complex simulations with recurrent
 connections this may no longer be the case.
 
+.. note::
+
+   Please note that MUSIC and Arbor are mutually exclusive and cannot be enabled at the same time.
```
Comment (jougs) [inline doc/tutorials/music_tutorial/music_tutorial_setlatency.rst:35]:
```suggestion
   Please note that MUSIC and the recording backend for Arbor are mutually exclusive and cannot be enabled at the same time. 
```

File: doc/tutorials/music_tutorial/music_tutorial_sli.rst [hunk 7]
```diff
@@ -103,4 +103,8 @@ in the introduciton is quite helpful, but the best resource is the set
 of example models in the NEST source code distribution. That will show
 you many useful idioms and typical ways to accomplish common tasks.
 
+.. note::
+
+   Please note that MUSIC and Arbor are mutually exclusive and cannot be enabled at the same time.
```
Comment (jougs) [inline doc/tutorials/music_tutorial/music_tutorial_sli.rst:108]:
```suggestion
   Please note that MUSIC and the recording backend for Arbor are mutually exclusive and cannot be enabled at the same time. 
```

File: doc/tutorials/music_tutorial/music_tutorial_tips.rst [hunk 8]
```diff
@@ -35,6 +35,11 @@ Start MUSIC using mpirun
     you try to process comand line options in your Pynest script, it is
     very likely you will confuse MUSIC.
 
+    .. note::
+
+       Please note that MUSIC and Arbor are mutually exclusive and cannot be enabled at the same time.
```
Comment (jougs) [inline doc/tutorials/music_tutorial/music_tutorial_tips.rst:40]:
```suggestion
       Please note that MUSIC and the recording backend for Arbor are mutually exclusive and cannot be enabled at the same time. 
```

File: pynest/examples/music_cont_out_proxy_example/nest_script.py [hunk 9]
```diff
@@ -28,6 +28,8 @@
 the NEST instances are observed by the music_cont_out_proxy and their
 values are forwarded through MUSIC to the receiver.
 
+Please note that MUSIC and Arbor are mutually exclusive and cannot be enabled at the same time.
```
Comment (jougs) [inline pynest/examples/music_cont_out_proxy_example/nest_script.py:31]:
```suggestion
Please note that MUSIC and the recording backend for Arbor are mutually exclusive and cannot be enabled at the same time. 
```

File: pynest/examples/music_cont_out_proxy_example/receiver_script.py [hunk 10]
```diff
@@ -24,6 +24,8 @@
 Music example receiver script
 ------------------------------
 
+Please note that MUSIC and Arbor are mutually exclusive and cannot be enabled at the same time.
```
Comment (jougs) [inline pynest/examples/music_cont_out_proxy_example/receiver_script.py:27]:
```suggestion
Please note that MUSIC and the recording backend for Arbor are mutually exclusive and cannot be enabled at the same time. 
```

**Comment by jougs** [comment]
Please also merge master and fix conflicts.

**Comment by sarakonradi** [comment]
@jougs No problem! The checks still seem to be failing, so this needs to be fixed.

**Comment by jougs** [comment]
Here's the output in a more readable form, just in case you can't find your way around:

```
MSGBLD0220: Python files with formatting errors:
MSGBLD0220: ... pynest/examples/music_cont_out_proxy_example/nest_script.py
    line 31, col 121: E501 line too long (121 > 120 characters)
    line 31, col 122: W291 trailing whitespace
MSGBLD0220: ... pynest/examples/music_cont_out_proxy_example/receiver_script.py
    line 27, col 121: E501 line too long (121 > 120 characters)
    line 27, col 122: W291 trailing whitespace
```

**Comment by sarakonradi** [comment]
Thanks @jougs, fixed the errors and the conflicts seem to be gone. :)

**Pull request merged**